src/main/java/com/clickhouse/kafka/connect/sink/ClickHouseSinkConfig.java

```diff
@@ -84,11 +83,38 @@ public ClickHouseSinkConfig(Map<String, String> props) {
         timeout = Integer.parseInt(props.getOrDefault(TIMEOUT_SECONDS, timeoutSecondsDefault.toString())) * MILLI_IN_A_SEC; // multiple in 1000 milli
         retry = Integer.parseInt(props.getOrDefault(RETRY_COUNT, retryCountDefault.toString()));
         exactlyOnce = Boolean.parseBoolean(props.getOrDefault(EXACTLY_ONCE,"false"));
-        sendProgressInHttpHeaders = Boolean.parseBoolean(props.getOrDefault(SEND_PROGRESS_IN_HTTP_HEADERS,"true"));
-        inputFormatSkipUnknownFields = Boolean.parseBoolean(props.getOrDefault(INPUT_FORMAT_SKIP_UNKNOWN_FIELDS,"true"));
         suppressTableExistenceException = Boolean.parseBoolean(props.getOrDefault("suppressTableExistenceException","false"));
+
+        Map<String, String> clickhouseSettings = new HashMap<>();
+        String clickhouseSettingsString = props.getOrDefault("clickhouseSettings", "").trim();
+
+        if (!clickhouseSettingsString.isBlank()) {
```

Comment:
Do we have the option to check if it is valid settings since if it is not valid we can log it to the user maybe he made a mistake in the setting (it is just an improvement to help us troubleshoot)